documentation/IDTA-01001/modules/ROOT/pages/includes/IDTA-01001_ValueOnly.adoc

```diff
@@ -48,6 +48,15 @@ The following rules shall be adhered to when serializing a submodel with the Ser
 
 * Submodel elements defined in the submodel other than the ones mentioned above are not subject to serialization of that SerializationModifier.
 
+
+
+The following rules shall be adhered to when serializing a submodel element different than a submodel with the SerializationModifier _Value_:
+
+* The submodel element is serialized as an unnamed JSON object.
```

Comment:
This statement is wrong, as to my understanding, a property should be serialized only as its value, e.g. `"some string property value"`. This is obviously not a JSON object but still valid JSON (see https://www.json.org/)